plugins/woocommerce/src/Admin/API/ShippingPartnerSuggestions.php

```diff
@@ -0,0 +1,212 @@
+<?php
+/**
+ * Handles requests for shipping partner suggestions.
+ */
+
+namespace Automattic\WooCommerce\Admin\API;
+
+use Automattic\WooCommerce\Admin\Features\ShippingPartnerSuggestions\DefaultShippingPartners;
+use Automattic\WooCommerce\Admin\Features\ShippingPartnerSuggestions\ShippingPartnerSuggestions as Suggestions;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * ShippingPartnerSuggestions Controller.
+ *
+ * @internal
+ * @extends WC_REST_Data_Controller
+ */
+class ShippingPartnerSuggestions extends \WC_REST_Data_Controller {
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc-admin';
+
+	/**
+	 * Route base.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'shipping-partner-suggestions';
+
+	/**
+	 * Register routes.
+	 */
+	public function register_routes() {
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => \WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_suggestions' ),
+					'permission_callback' => array( $this, 'get_permission_check' ),
+					'args'                => array(
+						'force_default_suggestions' => array(
+							'type'        => 'boolean',
+							'description' => __( 'Return the default shipping partner suggestions when woocommerce_show_shipping_partner_suggestions and woocommerce_setting_payments_recommendations_hidden options are set to no', 'woocommerce' ),
```

Comment:
Do you know where we set `woocommerce_show_shipping_partner_suggestions` options? And I guess we don't need `woocommerce_setting_payments_recommendations_hidden`?

Reply:
That description needs an update 😄 It only works when `woocommerce_show_marketplace_suggestions` is set to false and `force_default_suggestions` is set to true 

Code is here: https://github.com/woocommerce/woocommerce/pull/37155/files#diff-d0513d5e143fc2d4484715f024daa474107356bc224e4fa6e20bea70c04c03fdR97

Reply:
@chihsuan Updated in https://github.com/woocommerce/woocommerce/pull/37155/commits/66255379865608ba47583d47f3c41d032fefa83b 🙏